### DISCUSSION TITLE
Multiple user support

### DISCUSSION THREAD

**Discussion by HammyHavoc** [koreader/koreader#11162] [Ideas]
Would be great if multiple users could be a thing. A larger, high-end, family-shared e-ink device would make sense for use in the living room and encouraging people to read together. Likewise, would also be amazing if anyone could pick up an ereader in the house and access their own profile/user/books/progress.

To my knowledge, there isn't anything available that can do this, and KOReader is an interesting position to bring multiple users to previously single-user devices.

**Comment by NiLuJe** [comment]
This was touched upon in https://www.mobileread.com/forums/showthread.php?t=356833
